new/LOC6818Agurba.xml

```diff
@@ -64,10 +64,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </note>
                 </place>
                 <listRelation>
-                    <relation name="saws:formsPartOf"
+                    <relation name="dcterms:isPartOf"
                          active="LOC6818Agurba"
                          passive="LOC6817Zalan"/>
-                    <relation name="saws:isRelatedTo"
+                    <relation name="syriaca:has-relation-to-place"
```

Comment:
in the guidelines and here http://syriaca.org/documentation/relations.html, this is listed under person-place relations

Reply:
you are looking at an old commit